engines/wintermute/math/math_util.cpp

```diff
@@ -26,8 +26,12 @@
  * Copyright (c) 2011 Jan Nedoma
  */
 
-#include "engines/wintermute/math/math_util.h"
 #include "common/scummsys.h"
+#include "engines/wintermute/math/math_util.h"
```

Comment:
reorder include match scummvm?

Reply:
Ok, this confused me first. Should be correct now